addons/binding/org.openhab.binding.neeo/README.md

```diff
@@ -0,0 +1,289 @@
+# Neeo Binding
+
+This binding will help openHAB control a NEEO smart remote (specifically the NEEO Brain).
+NEEO is a smart home solution that includes an IP based remote.
+More information can be found at [NEEO](neeo.com) or in the forums at [NEEO Planet](https://planet.neeo.com).
+**This binding was not developed by NEEO** - so please don't ask questions on the NEEO forums.
```

Comment:
This is the case for 99% of all bindings. Unless there is any very strong reason why we have to state this here, please remove this line.

Reply:
This was a condition from NEEO itself when I started it all - they wanted me to make it quite clear that they are not the source of the binding/io and will not support it.